bots/notify/src/main/java/org/openjdk/skara/bots/notify/issue/IssueNotifier.java

```diff
@@ -311,9 +317,10 @@ public void onNewCommits(HostedRepository repository, Repository localRepository
                 if (!alreadyPostedComment) {
                     issue.addComment(commitNotification);
                 }
-                if (issue.state() == Issue.State.OPEN) {
+                if (issue.state() != Issue.State.OPEN) {
```

Comment:
Hi @erikj79 , I am wondering whether we need condition here. Do we only update the assignee when the issue is open?

Reply:
If it it not, I think I should update one assertion in IssueNotifierTests#testAltFixVersionsMatch

Reply:
oh, forgot to say, same thing here as in the other method. We don't need this conditional.